cmd/ceph-cosi-driver/cmd.go

```diff
@@ -78,7 +77,12 @@ func init() {
 }
 
 func run(ctx context.Context, args []string) error {
-	identityServer, bucketProvisioner, err := driver.NewDriver(ctx, provisionerName)
+	driverPrefix := os.Getenv("DRIVER_PREFIX")
+	if driverPrefix == "" {
+		driverPrefix = os.Getenv("POD_NAMESPACE")
```

Comment:
Is `POD_NAMESPACE` a magic env var that is present in all k8s pods, or is this something that needs to be set via a pod's downward API?

Reply:
It was not exposed properly for the driver, now it exposed as Enviroment Variable for driver